tools/model_explorer_circle/MANIFEST.in

```diff
@@ -0,0 +1,3 @@
+graft src
+
+global-exclude *~ *.py[cod]
```

Comment:
from doc, https://setuptools.pypa.io/en/latest/userguide/miscellaneous.html#controlling-files-in-the-distribution
> the contents of the directory tree src will first be added to the sdist, and then after that all files in the sdist with a .pyc, .pyo, or .pyd extension will be removed from the sdist